cs251tk/toolkit/__main__.py

```diff
@@ -125,7 +131,14 @@ def main():
                     print_progress(result['username'])
                     results.append(result)
                     records.extend(recording)
-
+        elif web:
+            Thread(target=run_server, args=(basedir,), daemon=True).start()
+            for student in usernames:
+                print("\nStudent: {}".format(student))
```

Comment:
Print and logging? Can we remove the print?

Reply:
The print is to specify when the toolkit moves on to the next student. We can remove the logging.